### DISCUSSION TITLE
Lower the video resolution based on your network in edumeet?

### DISCUSSION THREAD

**Discussion by prayaslashkari** [edumeet/edumeet#882] [Ideas]
I am trying to implement this feature, which lowers the video resolution based on your network profile.

As noticed in average call with lastN 12, and total participants around 40. It uses up my 2gb of mobile data.

Any suggestions how to tackle this?

**Comment by vpalmisano** [comment]
The automatic switching between one simulcast video layer and another is regulated by the mediasoup library. If you what to improve the actual mechanism, you have to look inside this: https://github.com/versatica/mediasoup/blob/v3/worker/src/RTC/SimulcastConsumer.cpp#L379
